helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -1705,4 +1734,27 @@ private static void verifyToken(final String token, final Job job)
     }
   }
 
+  private List<TaskStatus.State> getPreviousJobStates(final JobId jobId, final String host) {
+    List<TaskStatus.State> previousStates;
```

Comment:
Can this be `final` since its assignment is the last statement in the `try` below? Not sure. Java... how does it work?